types/jest-global.d.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference types="expect-webdriverio/types/standalone"/>
```

Comment:
Please revert the change, this would break types

Reply:
Done!